docs/upstream-differences.md

```diff
@@ -15,7 +16,9 @@ upstream OCP.
     git remote add upstream https://github.com/openshift/installer.git
     git fetch upstream -a
     ```
-1. See carry patches from previous release:
+
+2. See carry patches from previous release:
```

Comment:
>It is sort of a standard since the actual numbering in markdown is always ignored when it's rendered, and the rendered order is readjusted to that of appearance. It also prevents having to manually change all the lines again in the future to keep consistency if new steps are added or removed (even if it's ignored when rendering, numbering inconsistencies can cause confusion to an editor).

I didn't understand/acknowledge this logic, but I do get it now.